RELEASE/scripts/autoscend/auto_bondmember.ash

```diff
@@ -634,10 +640,6 @@ boolean LM_bond()
 		{
 			if((item_amount($item[Filthworm Royal Guard Scent Gland]) == 0) && (item_amount($item[Heart Of The Filthworm Queen]) == 0) && canYellowRay())
 			{
-				while((get_property("timesRested").to_int() < total_free_rests()) && (my_mp() < mp_cost($skill[Disintegrate])))
```

Comment:
they already do an acquireMP check below this, so just defer to that and let the restoration logic work out how to get there